packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -502,6 +498,36 @@ export default function SettingsLayout({
   );
 }
 
+const SidebarContainerElement = ({
+  sideContainerOpen,
+  bannersHeight,
+  setSideContainerOpen,
+}: SidebarContainerElementProps) => {
+  const { t } = useLocale();
+  return (
+    <>
+      {/* Mobile backdrop */}
+      {sideContainerOpen && (
+        <button
+          onClick={() => setSideContainerOpen(false)}
+          className="fixed left-0 top-0 z-10 h-full w-full bg-black/50">
+          <span className="sr-only">{t("hide_navigation")}</span>
+        </button>
+      )}
+      <SettingsSidebarContainer
+        navigationIsOpenedOnMobile={sideContainerOpen}
+        bannersHeight={bannersHeight}
+      />
+    </>
+  );
+};
+
+type SidebarContainerElementProps = {
+  sideContainerOpen?: boolean;
+  bannersHeight?: number;
+  setSideContainerOpen?: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
this is causing type error use

setSideContainerOpen: React.Dispatch<React.SetStateAction<boolean>>;

Reply:
Did the changes now. Thanks @Udit-takkar